scripts/Thaumcraft-07-Aspect-Adds.zs

```diff
@@ -6,6 +6,16 @@
 
 
 
+// --- Minecraft ---
+
+
+// --- Book
+mods.thaumcraft.Aspects.set(<minecraft:book>, "pannus 1, cognitio 1");
+
+// --- Enchanted book
+mods.thaumcraft.Aspects.set(<minecraft:enchanted_book>, "pannus 1, cognitio 1, metallum 1");
+
```

Comment:
Is missing 2×`volatus` and 2×`praecantatio`

Should be:
```zs
mods.thaumcraft.Aspects.set(<minecraft:enchanted_book>, "pannus 1, cognitio 1, volatus 2, praecantatio 2, metallum 1");
```
```

Reply:
It also gets aspects from the enchantments it has on it, if you look at a creatively spawned one with no enchantment, it has only the three I added.

Reply:
ok then